### ISSUE TITLE
Icerec parasitic build Python version mismatch (Trac #2242)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2234]
http://software.icecube.wisc.edu/documentation/projects/cmake/parasite.html
I am following this tutorial for making a parasite metaproject. 
I have run eval `/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/setup.sh`. This is the python that I would like to use. 
My cmake call is cmake ../src -DMETAPROJECT=icerec/V05-02-00 -DCMAKE_INSTALL_PREFIX=icerec-plus.${OS_ARCH} . 
This make'd fine. When I run ./env-shell.sh to enter into the environment provided by the parasite, I get 

****************************************************************
Python version mismatch found:
IceTray was compiled with
Currently running with    2.7.13
****************************************************************
Environment not (re)loaded.

I have run through these steps, and you can see the result in /data/user/erixencruz/software/meta-projects/icerec/scrap/ .

In /data/user/erixencruz/software/meta-projects/icerec/V05-02-00/ I have a workaround where I simply commented out the python version check in build/env-shell.sh . This allows me to run the shell script and enter the environment.

Thanks,
Erixen Cruz
erixen.cruz@icecube.wisc.edu

Migrated from https://code.icecube.wisc.edu/ticket/2242
```json
{
    "status": "closed", 
    "changetime": "2019-02-26T14:09:09", 
    "description": "http://software.icecube.wisc.edu/documentation/projects/cmake/parasite.html\nI am following this tutorial for making a parasite metaproject. \nI have run eval `/cvmfs/icecube.opensciencegrid.org/py2-v3.0.1/setup.sh`. This is the python that I would like to use. \nMy cmake call is cmake ../src -DMETAPROJECT=icerec/V05-02-00 -DCMAKE_INSTALL_PREFIX=icerec-plus.${OS_ARCH} . \nThis make'd fine. When I run ./env-shell.sh to enter into the environment provided by the parasite, I get \n\n****************************************************************\nPython version mismatch found:\nIceTray was compiled with\nCurrently running with    2.7.13\n****************************************************************\nEnvironment not (re)loaded.\n\n\nI have run through these steps, and you can see the result in /data/user/erixencruz/software/meta-projects/icerec/scrap/ .\n\nIn /data/user/erixencruz/software/meta-projects/icerec/V05-02-00/ I have a workaround where I simply commented out the python version check in build/env-shell.sh . This allows me to run the shell script and enter the environment.\n\nThanks,\nErixen Cruz\nerixen.cruz@icecube.wisc.edu", 
    "reporter": "icecube", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1551190149662003", 
    "component": "analysis", 
    "summary": "Icerec parasitic build Python version mismatch", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2019-02-26T12:20:36", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190226T13:51:02: kjmeagher commented:

> This is caused by a backwards incompatibility between the trunk of cmake used to create the parasitic meta-project and the original release meta-project. I don't know any way to solve this issue completely.
> 
> As a work around can you redo the instructions but this time add `-r3130` to the `svn co ...` line?

**Comment by icecube-trac** [comment]
Trac update at 20190226T14:09:09: 
* jvansanten changed owner from "" to "jvansanten"
* jvansanten commented:

> In changeset 171575:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171575"
> Add PYTHON_VERSION for parasitic builds against old metaprojects. Fixes #2234.
> 
> Along the way, point suggested cmake URL to combo/trunk
> ```


* jvansanten changed resolution from "" to "fixed"
* jvansanten changed status from "new" to "closed"
